src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -424,7 +467,123 @@ static void Main()
 }";
             string output = @"bcfgjklln";
 
-            CompileAndVerify(source: source, expectedOutput: output);
+            CompileAndVerify(source: source, expectedOutput: output)
+                .VerifyOperationTree(@"
```

Comment:
Yes, this is just a convenience static helper that grabs the _compilation field from the CompilationVerifier and invokes `compilation.VerifyOperationTree`.